src/main/java/org/team1540/robot2024/subsystems/vision/AprilTagVisionIO.java

```diff
@@ -0,0 +1,25 @@
+package org.team1540.robot2024.subsystems.vision;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import org.littletonrobotics.junction.AutoLog;
+
+public interface AprilTagVisionIO {
+    @AutoLog
+    class AprilTagVisionIOInputs {
+        public Pose2d estimatedPoseMeters = new Pose2d();
+        public boolean hasTarget = false;
```

Comment:
This is never used

Reply:
I added this because I thought it would be useful for logging, but yeah it doesn't seem necessary since we have arrays of all of the tags being seen.